mautic/mautic
# Php 8.2  with new CS Fixer version

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [y]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

~~⚠️ this PR is based on https://github.com/mautic/mautic/pull/12469 so it must be merged first!~~

I'm pushing these changes here because I want to separate manual (https://github.com/mautic/mautic/pull/12469) changes with the automatic changes in this PR.

The reason for upgrade of the CS Fixer library is that on PHP 8.2 it throws this error:
```
composer fixcs
> bin/php-cs-fixer fix
PHP needs to be a minimum version of PHP 7.4.0 and maximum version of PHP 8.1.*.
Current PHP version: 8.2.14.
To ignore this requirement please set `PHP_CS_FIXER_IGNORE_ENV`.
If you use PHP version higher than supported, you may experience code modified in a wrong way.
Please report such cases at https://github.com/PHP-CS-Fixer/PHP-CS-Fixer .
Script bin/php-cs-fixer fix handling the fixcs event returned with error code 1
```
There is a walk around solution suggested in the error message but I think it's time to update the CS Fixer. This PR contains only the changes generated by the new CS Fixer version.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Click through to test that nothing was broken. But we have automatic tests that would discover issues if pages wouldn't load so I think this doesn't require any manual testing.
